### ISSUE TITLE
`get_tree_plot_for_tissue` relies on ggraph version 2.1.0

### ISSUE THREAD

**Issue by christopherjin** [MoTrPAC/MotrpacRatTraining6mo#55]
Hi folks,

Went through a few testing iterations because I just couldn't get `get_tree_plot_for_tissue` to work at all. I was using ggraph v 2.2.1, which gives the following error. Re-installing to v 2.1.0 fixed the issue

Error message for 2.2.1
```{r}
Error in `[<-`:
! Assigned data `d_g_our_layout[grepl("F0_M0", d_g_our_layout$name), "y"][1]` must be compatible with row subscript `d_g_our_layout$name ==
  "0w"`.
✖ 1 row must be assigned.
✖ Assigned data has 4 rows.
ℹ Row updates require a list value. Do you need `list()` or `as.list()`?
Caused by error in `vectbl_recycle_rhs_rows()`:
! Can't recycle input of size 4 to size 1.
Backtrace:
  1. MotrpacRatTraining6mo::get_tree_plot_for_tissue(...)
 14. tibble (local) `<fn>`(`<vctrs___>`)
 15. tibble:::vectbl_recycle_rhs_rows(value, length(i), i_arg, value_arg, call)
 ```

**Comment by nicolerg** [comment]
Thanks, @christopherjin. I'll look into making it compatible with `ggraph 2.2.1`

**Comment by christopherjin** [comment]
Thank you Nicole!

**Comment by simplefywoo** [comment]
Wow, it turns out that the version of this package conflicts, and it is really ok after replacing the version. I also thought that I did not use the get_tree_plot_for_tissue function correctly.

**Comment by nicolergay** [comment]
Thanks for confirming. I am away the next few weeks but plan on addressing it when I get back.

**Comment by nicolerg** [comment]
Apologies for the protracted delay. This is now fixed in v1.6.6
